modules/plugin/jdbc/jdbc-oracle/src/test/java/org/geotools/data/oracle/Oracle3DOnlineTest.java

```diff
@@ -32,5 +33,28 @@ protected JDBC3DTestSetup createTestSetup() {
      * GEOT-4133
      */
     @Override
-    public void testCreateSchemaAndInsertPolyRectangle() throws Exception {}
+    public void testCreateSchemaAndInsertPolyRectangle() {}
+
+    /**
+     * This test does not work in Oracle, not even in SQL.
```

Comment:
Can we correct the test? JTS does now support the Z and M parameters so it should finally be possible to make this work.

Reply:
I'll take another look at this, I think `SDOSqlDumper` only ever outputs 2D SDO geometries; but I'm not sure that is used or that `GeometryConverter` is used which should output 3D geometries (apart from Rectangle which seems a requirement for SDO filters - this may of course have changed), see:

https://github.com/geotools/geotools/blob/ff084efa09788eb08f27330c0cc8a46829e99584/modules/plugin/jdbc/jdbc-oracle/src/main/java/org/geotools/data/oracle/sdo/GeometryConverter.java#L161-L182